OADB/macros/BrowseAndFillPhysicsSelectionOADB.C

```diff
@@ -622,12 +639,12 @@ AliOADBPhysicsSelection* Pbp2016(char* name = "oadb_Pbp_2016"){
   return oadb_Pbp_2016;
 }
 
-AliOADBPhysicsSelection* pp2010(char* name = "oadbLHCpp2010"){
+AliOADBPhysicsSelection* pp2010(const char* name){
   // pp2010-11
   AliOADBPhysicsSelection * oadbLHCpp2010 = new AliOADBPhysicsSelection(name);
   UInt_t triggerCount = 0;
   oadbLHCpp2010->AddCollisionTriggerClass(AliVEvent::kINT1,"+CINT1B-ABCE-NOPF-ALL,CINT1-B-NOPF-ALL[NOTRD|]","B",triggerCount);
-  oadbLHCpp2010->AddCollisionTriggerClass(AliVEvent::kINT1 | AliVEvent::kFastOnly,"+CINT1-B-NOPF-FASTNOTRD -CINT1-B-NOPF-ALLNOTRD","B",triggerCount);
+  oadbLHCpp2010->AddCollisionTriggerClass((AliVEvent::EOfflineTriggerTypes)(AliVEvent::kINT1 | AliVEvent::kFastOnly),"+CINT1-B-NOPF-FASTNOTRD -CINT1-B-NOPF-ALLNOTRD","B",triggerCount);
```

Comment:
Explicit cast is necessary in ROOT6 as rvalue (AliVEvent::kINT1 | AliVEvent::kFastOnly) is int. However in ROOT5 the cast should have been done explicitly, so the logics should not have changed.